<!-- -*- mode: xml -*- -->
<launch>
  <node name="range" pkg="swissranger_camera" type="swissranger_camera"
        respawn="false" output="screen" clear_params="true">
    <param name="frame_id" type="string" value="/SR4000"/>
    <!--
    <param name="camera_info_url"
           value="file:///$(find cr_capture)/launch/sr4000.yaml"/>
    -->
    <!-- <remap from="~distance/image_raw16" to="~image_depth"/> -->
  </node>
  <!--
  <node name="iview" pkg="image_view" type="image_view">
    <remap from="image" to="sr4000/intensity/image_raw"/>
  </node>
  -->
</launch>
